tests/utils.sh

```diff
@@ -376,18 +376,27 @@ function assert_no_line_match()
 # @result_to_compare Raw output to be compared
 function assert_equals_helper()
 {
+  # colors to use to print error
+  local _none='\033[0m'
+  local _red='\033[1;31m'
+  local _blue='\033[1;34m'
+
   local msg="$1"
   local line="$2"
-  # See bugs section in github.com/koalaman/shellcheck/wiki/SC2178
-  # shellcheck disable=SC2178
   local expected="$3"
   local result_to_compare="$4"
 
   line=${line:-'Unknown line'}
 
-  # See bugs section in github.com/koalaman/shellcheck/wiki/SC2178
-  # shellcheck disable=2128
-  assertEquals "line $line: $msg" "$result_to_compare" "$expected"
+  if ! assertEquals "$expected" "$result_to_compare" &> /dev/null; then
+    printf '%bASSERT:%b line %s: %s
+%bExpected Result:%b %b%s%b 
+%b-> Actual Result:%b %b%s%b\n' \
```

Comment:
Add two extra spaces, something like this:
```
ASSERT: line (372): Default ARCH did not match expectation
  Expected Result: x86_4 
  -> Actual Result: x86_66
```

Reply:
Also, could you keep the first parameter of `printf` in the same line?

Reply:
I thought it would be more readable. I saw it in compare_command_sequence<br>  Changed it now to be in one line.